pipeline/metadata/6.0.yaml

```diff
@@ -635,6 +635,22 @@ suites:
       - rgw
       - stage-1
 
+  - name: "Testing RGW STS, DBR and other Tier-1"
+    suite: "suites/quincy/rgw/tier-1-extn_rgw.yaml"
+    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
```

Comment:
This perticular suite has all the test cases from pacific/tier-1-extn_rgw.yaml and additional.
So instead of having new entry in 6.0.yaml, can we make changes to existing entry?
    suite: "suites/quincy/rgw/tier-1-extn_rgw.yaml"
    global-conf: "conf/quincy/rgw/tier-0_rgw.yaml"
    In https://github.com/red-hat-storage/cephci/blob/master/pipeline/metadata/6.0.yaml#L494

Reply:
@anrao19 The Rate limiting tier-1 tests in this suite is from quincy only , so I cannot add the same to the pacific suite , the NFS test can be added though. So I thought creating a new suite file was a good idea. If there is a better way around this I am open to discussion.

Reply:
@TejasC88, About adding a new suite under quincy folder was a good idea. I was just asking the changes in metadats file.

In 6.0 metadata file we are already having entry for tier-1-extn_rgw.yaml, we can just change its path from pacific to quincy
suite: "suites/quincy/rgw/tier-1-extn_rgw.yaml"
global-conf: "conf/quincy/rgw/tier-0_rgw.yaml"
In line https://github.com/red-hat-storage/cephci/blob/master/pipeline/metadata/6.0.yaml#L494

Reply:
@anrao19 I have done the changes like you suggested , thanks , request you to review now?

Reply:
LGTM